syntax = "proto3";

package tensorflow;

option go_package = "github.com/tensorflow/tensorflow/tensorflow/go/core/protobuf/for_core_protos_go_proto";

// Extra data needed on a non-RDMA RecvBufResponse.
message RecvBufRespExtra {
  repeated bytes tensor_content = 1;
}
